.travis.yml

```
@@ -1,15 +1,18 @@
 language: php
 
 php:
-  - 7.0
-  - 7.1
   - 7.2
+  - 7.3
+  - 7.4
 
 before_install:
   - sudo apt-get -qq update
 
 install:
-  - composer self-update
+  - php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
+  - php -r "if (hash_file('sha384', 'composer-setup.php') === '756890a4488ce9024fc62c56153228907f1545c228516cbf63f885e036d37e9a59d27d63f46af1d4d07ee0f76181c7d3') { echo 'Installer verified'; } else { echo 'Installer corrupt'; unlink('composer-setup.php'); } echo PHP_EOL;"
+  - php composer-setup.php --version=1.10.19
```

```suggestion
  - php composer-setup.php --1
```

Why do we fix this to a specific version? I suggest using at least the latest v1 version. Or can we even switch to v2?

---

I've been using this with Composer 2 just fine, so it probably could do without a version

---

Indeed, composer 2 is now supported on both 2.3 and 2.4, so I agree that there's no need to hardcore composer version to v1